src/main/resources/application.yml

```diff
@@ -27,6 +29,10 @@ application:
     base-url: http://NOTIFICATION
   booking:
     base-url: http://BOOKING
+  auth:
+    base-url: http://localhost:8082
```

Comment:
User service names instead of localhost and ports.

Reply:
Resolved